src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <Age age={person.age} />
+    <Partner
+      isMarried={person.isMarried}
+      sex={person.sex}
+      partnerName={person.partnerName}
+    />
+  </section>
+);
+
+const Age = ({ age }) => age && <p className="Person__age">{`I am ${age}`}</p>;
+const Partner = ({ isMarried, sex, partnerName }) => {
+  let message = 'I am not married';
+
+  if (isMarried && sex === 'm') {
+    message = `${partnerName} is my wife`;
+  }
+
+  if (isMarried && sex === 'f') {
+    message = `${partnerName} is my husband`;
+  }
+
+  return <p className="Person__partner">{`${message}`}</p>;
+};
```

Comment:
If `isMarried` is false - we know for sure that we should return `I am not married` and we don't need to do further checks

The same is true for `isMarried && sex === 'm'`— we already know what we need to return and the `message` won't be reassigned, since the conditions exclude each other.

In the current code version, we run all the checks no matter what. I suggest to think of a way to avoid `let` here and return right away